doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Timers in node.js
+layout: docs.hbs
+---
+
+# Timers in node.js and beyond
+
+Timers are a collection of global functions in node.js that allow you to 
+execute code after a set period of time. To fully understand when timer 
+functions will be executed, it's a good idea to read up on the the node 
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
+
+## Code Time Machine
+
+The node API provides several ways to schedule your code to execute at 
+some point after the present moment.
+
+### "When I say so" Execution ~ *setTimeout()*
+
+Using `setTimeout()`, you can schedule code to execute after a designated 
+amount of milliseconds. You may already be familiar with this function, as 
+it is a part of V8 and thus part of the browser JavaScript API.
```

Comment:
> it is a part of V8

Is it really? ;)

I'm sure many people don't know (and that is ok! It means we have done a good job), but we actually need to have our own timers implementation. Browsers provide that, not the JS Engine. I'll add an extra note in the main comment thread here.